setup-ubuntu-14-lts.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# This setup script is inspired by Scala Native's setup script:
+# https://github.com/scala-native/scala-native/blob/master/bin/travis_setup.sh
+
+# Fail if command failed, variable is unassigned, also show executed commands
+set -eux
+
+sudo apt-get update -qq
+
+echo "Remove libunwind pre-bundled with clang"
+sudo find /usr -name "*libunwind*" -delete
```

Comment:
This worries me a bit - is there any way we can do it with the package manager? I'd be wary of deleting files on a user's machine. If this script is only going to be used in Travis then it's less worrisome but then perhaps we should name the script as such.

Reply:
Yes, you are right.

I think for now I have removed the Ubuntu 14 LTS setup, and I have created a Travis specific setup script.

I'd like to attempt the Ubuntu 14 setup in a separate PR, since I think it's still worthwhile to get the Travis build in - wdyt?

Reply:
That sounds good to me.